src/reset-password/data/service.js

```diff
@@ -27,15 +27,14 @@ export async function resetPassword(payload, token, queryParams) {
     headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
     isPublic: true,
   };
+  const url = new URL(`${getConfig().LMS_BASE_URL}/password/reset/${token}/`);
```

Comment:
I dont think we use this query param, Just for parity sake. We can remove it in future if we need. 
```suggestion
  const url = new URL(`${getConfig().LMS_BASE_URL}/password/reset/${token}/?track=pwreset`);
```

Reply:
Since this is already part of the password reset URL, we don't need to set it on the API endpoint as well. This query param is just used to track the password reset page in google analytics.